libpod/boltdb_state.go

```diff
@@ -408,12 +408,29 @@ func (s *BoltState) ValidateDBConfig(runtime *Runtime) error {
 	}
 	defer s.deferredCloseDBCon(db)
 
+	//runRootName := "storage temporary directory (runroot)"
+	//runRootKey := []byte(runRootName)
 	// Check runtime configuration
 	if err := checkRuntimeConfig(db, runtime); err != nil {
+		/*	if runtime.newDB {
```

Comment:
Why is this commented code here?  If it is not going to be used, remove it.  Git will remember the code.